src/SelectedWallet/WalletSelection/WalletSelectionScreen.tsx

```diff
@@ -157,8 +159,8 @@ const ShelleyButton = () => {
         navigation.navigate('new-wallet', {
           screen: 'choose-create-restore',
           params: {
-            networkId: CONFIG.NETWORKS.HASKELL_SHELLEY.NETWORK_ID,
-            walletImplementationId: CONFIG.WALLETS.HASKELL_SHELLEY.WALLET_IMPLEMENTATION_ID,
+            networkId: HASKELL_SHELLEY.NETWORK_ID,
```

Comment:
CONFIG.NETWORKS is based on the build, `.env.{production,staging,nightly}` `USE_TESTNET` -> this will cause staging (dev) to only add mainnet wallet, I think we should add more Buttons in dev. and keep this way for prod.